### ISSUE TITLE
[REVIEW]: biopeaks: a graphical user interface for feature extraction from heart- and breathing biosignals

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2621]
**Submitting author:** @JanCBrammer (<a href="http://orcid.org/0000-0002-7664-3753">Jan C. Brammer</a>)
**Repository:** <a href="https://github.com/JanCBrammer/biopeaks" target ="_blank">https://github.com/JanCBrammer/biopeaks</a>
**Version:** v1.4.1
**Editor:** @oliviaguest
**Reviewer:** @TomDonoghue, @sappelhoff, @jamesheathers
**Archive:** <a href="https://doi.org/10.5281/zenodo.4133324" target="_blank">10.5281/zenodo.4133324</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/610ce9dc258bcf3d5ccd311e349b946a/status.svg)](https://joss.theoj.org/papers/610ce9dc258bcf3d5ccd311e349b946a)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/610ce9dc258bcf3d5ccd311e349b946a"><img src="https://joss.theoj.org/papers/610ce9dc258bcf3d5ccd311e349b946a/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/610ce9dc258bcf3d5ccd311e349b946a/status.svg)](https://joss.theoj.org/papers/610ce9dc258bcf3d5ccd311e349b946a)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/JanCBrammer/biopeaks" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@TomDonoghue & @sappelhoff & @jamesheathers, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @oliviaguest know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JanCBrammer/biopeaks">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@JanCBrammer) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems). 
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JanCBrammer/biopeaks">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@JanCBrammer) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed? https://github.com/JanCBrammer/biopeaks/issues/14#issue-694096081
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution. https://github.com/JanCBrammer/biopeaks/pull/13
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems). https://github.com/JanCBrammer/biopeaks/issues/12
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)? https://github.com/JanCBrammer/biopeaks/issues/11 https://github.com/JanCBrammer/biopeaks/issues/10
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified? https://github.com/JanCBrammer/biopeaks/issues/8
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support https://github.com/JanCBrammer/biopeaks/issues/9

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)? https://github.com/JanCBrammer/biopeaks/pull/6

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JanCBrammer/biopeaks">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@JanCBrammer) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @TomDonoghue, @sappelhoff, @jamesheathers it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.5281/ZENODO.3597887 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.1109/MCSE.2011.37 is OK
- 10.1016/0013-4694(92)90009-7 is OK
- 10.1080/03091902.2019.1640306 is OK
- 10.3758/s13428-011-0107-7 is OK
- 10.1088/0967-3334/37/7/N38 is OK
- 10.1111/1469-8986.3510127 is OK
- 10.1371/journal.pone.0076585 is OK
- 10.1088/1361-6579/aad7e6 is OK

MISSING DOIs

- 10.25080/majora-92bf1922-00a may be a valid DOI for title: Data Structures for Statistical Computing in Python

INVALID DOIs

- https://doi.org/10.1038/s41592-019-0686-2 is INVALID because of 'https://doi.org/' prefix
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02621/joss.02621/10.21105.joss.02621.pdf)

**Comment by oliviaguest** [comment]
👋👋👋

Hey @JanCBrammer @TomDonoghue, @sappelhoff, @jamesheathers! Welcome to where the review itself will take place. Please make sure to read the instructions above.

For any and all things worthy of discussion or comment, use this issue right here — so drop comments or questions for me, the author, etc., here. For any very code-specific things please feel free to start an issue on the repo of the code itself (if appropriate!) and link back to it from here. For an example of how this process plays out feel free to skim previous reviews, such as: https://github.com/openjournals/joss-reviews/issues/2285 and https://github.com/openjournals/joss-reviews/issues/2348. ☺️

**Comment by JanCBrammer** [comment]
@sappelhoff, thanks for your thorough and helpful review so far, much appreciated! I'm offline next week (week 38), so I won't get around to address your questions and suggestions until the week after the next (week 39). Looking forward to it :)

**Comment by oliviaguest** [comment]
Hey, no super pressure but when you (@TomDonoghue, @sappelhoff, @jamesheathers) get a chance could you give us a rough estimate for an ETA on your reviews, please? ☺️

**Comment by sappelhoff** [comment]
My final evaluation is on hold until @JanCBrammer has addressed or replied to some of my remarks. But other than that, I am basically done reviewing - see the linked issues and prs. :-)

**Comment by oliviaguest** [comment]
@sappelhoff oh, amazing! Great work — thanks. 👍

**Comment by TomDonoghue** [comment]
I should be able to do my review by the end of the week!

**Comment by oliviaguest** [comment]
@TomDonoghue Oh, great! I'll set up a reminder for both of us.

**Comment by oliviaguest** [comment]
@whedon remind @TomDonoghue in 9 days

**Comment by whedon** [comment]
Reminder set for @TomDonoghue in 9 days

**Comment by sappelhoff** [comment]
As an update: @JanCBrammer has addressed most of my suggestions by now. There are three remaining open issues from my side:

- consistent docstring documentation
- test coverage reporting
- developer documentation on how to build the docs

It'd be nice to have these points addressed before a final evaluation.

**Comment by TomDonoghue** [comment]
Just a note that I have basically finished my first round of review. Thanks to @sappelhoff for making my job easier, as he already pointed out a lot of good suggestions! I made some additional suggestions, mostly by opening a couple issues on the tool repo. As a general note, I consider a few of the issues I opened as _suggestions_ (not strictly required for the paper), and so to guide what I think is most important would be:
- [ ] I agree with @sappelhoff in particular on fixing up the documentation a bit (https://github.com/JanCBrammer/biopeaks/issues/11)
- [ ] some small paper checks & fixes (https://github.com/JanCBrammer/biopeaks/issues/16)
- [ ] noting test dependencies and how to run them, which I've added to an existing issue (https://github.com/JanCBrammer/biopeaks/issues/8)

The other issues are potential clean ups / clarifications / extensions that may be useful for the module, but are perhaps a little longer term, and I don't think necessarily need to be done now.

When these things are all checked / done, I'm happy to come back and sign-off on the remaining points.

**Comment by JanCBrammer** [comment]
@sappelhoff, @TomDonoghue, thank you for you clear and thorough reviews! I'll try to address your issues over the coming two weeks. I'll keep you in the loop by tagging you once I've resolved an issue. Also, I will notify you here once I think the repo is ready for your final evaluation / sign-off (in case you don't have time and/or resources to keep track of single issues).

**Comment by whedon** [comment]
:wave: @TomDonoghue, please update us on how your review is going.

**Comment by JanCBrammer** [comment]
@sappelhoff, @TomDonoghue, I've addressed all "blocking" issues (JanCBrammer/biopeaks#8, JanCBrammer/biopeaks#10, JanCBrammer/biopeaks#11, JanCBrammer/biopeaks#16) and the repository is now ready for your final evaluation.

**Comment by oliviaguest** [comment]
@jamesheathers please give us an ETA if you can here. ☺️

**Comment by sappelhoff** [comment]
I checked all issues that I raised with @JanCBrammer and everything has been addressed to my full satisfaction. `biopeaks` is a useful piece of software that will make life easier for its users. The accompanying paper is well written, the documentation is up to date an thorough. Unit tests and coverage reports are in place, and best practices of open source software (and open science, where applicable) are being followed. I see a need for software like `biopeaks` that provides a GUI for users that are not (yet) familiar with scripting analyses.

I recommend accepting this project for a publication in JOSS.

**Comment by jamesheathers** [comment]
Hello all,

My apologies for the delay. There was also another problem: I can't find anything to change here.

* I got this to run cleanly and straightforwardly with a fresh install of Miniconda
* All of the core functionality I would expect for a R-peak/systolic interval detector is present, and it all works to a higher reliability than simpler peak detection methods
* I gave it an engineer (not Python naive but biosignal analysis naive) and he was immediately able to get the software to function as intended and to my satisfaction
* Any additional suggestions for analytic functions I could make would go beyond the scope of the paper
* The documentation is extremely good

In short, I can't actually find anything I'd want to change that makes a reasonable contribution here. I don't think I've ever written an exclusively positive review before, but here we are. This will form a useful tool for beginner to intermediate biosignal analysts who wish to explore cardiac cyclic time series.

**Comment by oliviaguest** [comment]
@JanCBrammer awesome, I think all reviewers are happy now. Can you please deposit the code on zenodo or similar and post the DOI back here? ☺️

**Comment by JanCBrammer** [comment]
@oliviaguest, I released a new version ([v.1.4.1](https://github.com/JanCBrammer/biopeaks/releases/tag/v1.4.1)) containing the improvements related to this review. I've then deposited the repository at [10.5281/zenodo.4133324](http://doi.org/10.5281/zenodo.4133324).

**Comment by oliviaguest** [comment]
@whedon set 10.5281/zenodo.4133324 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.4133324" target="_blank">10.5281/zenodo.4133324</a> is the archive.

**Comment by oliviaguest** [comment]
@whedon set v1.4.1 as version

**Comment by whedon** [comment]
OK. v1.4.1 is the version.

**Comment by oliviaguest** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02621/joss.02621/10.21105.joss.02621.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02621/joss.02621/10.21105.joss.02621.pdf) :page_facing_up: :point_left:

**Comment by oliviaguest** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.5281/ZENODO.3597887 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.1016/0013-4694(92)90009-7 is OK
- 10.1080/03091902.2019.1640306 is OK
- 10.3758/s13428-011-0107-7 is OK
- 10.1088/0967-3334/37/7/N38 is OK
- 10.1111/1469-8986.3510127 is OK
- 10.1371/journal.pone.0076585 is OK
- 10.1088/1361-6579/aad7e6 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by oliviaguest** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by oliviaguest** [comment]
@JanCBrammer does this all look good to you (the final PDF, etc.)?

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.5281/ZENODO.3597887 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.1016/0013-4694(92)90009-7 is OK
- 10.1080/03091902.2019.1640306 is OK
- 10.3758/s13428-011-0107-7 is OK
- 10.1088/0967-3334/37/7/N38 is OK
- 10.1111/1469-8986.3510127 is OK
- 10.1371/journal.pone.0076585 is OK
- 10.1088/1361-6579/aad7e6 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1863

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1863, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by JanCBrammer** [comment]
> does this all look good to you (the final PDF, etc.)?

@oliviaguest, the final article proof looks good to me!

**Comment by arfon** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1321190653318017024) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1872
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02621](https://doi.org/10.21105/joss.02621)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by arfon** [comment]
@TomDonoghue, @sappelhoff, @jamesheathers - many thanks for your reviews here and to @oliviaguest for editing this submission ✨ 

@JanCBrammer - your paper is now accepted into JOSS :zap::rocket::boom:

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02621/status.svg)](https://doi.org/10.21105/joss.02621)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02621">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02621/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02621/status.svg
   :target: https://doi.org/10.21105/joss.02621
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02621/status.svg)](https://doi.org/10.21105/joss.02621)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss

**Comment by JanCBrammer** [comment]
@oliviaguest, @TomDonoghue, @sappelhoff, @jamesheathers thank you all for your time and the thorough, helpful and timely reviews!
